.github/workflows/ci.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby: ['2.4', '2.5', '2.6', '2.7']
+        ruby: ['2.4', '2.5', '2.6', '2.7', 'head']
```

Comment:
should be marked as allow-failure since it's a moving target

```
continue-on-error: ${{ matrix.experimental }}
strategy:
  matrix:
    ruby: ['2.4', '2.5', '2.6', '2.7']
    experimental: [false]
    include:
    - ruby: head
      experimental: true
```